src/node/rpc/frontend.h

```diff
@@ -552,15 +570,57 @@ namespace ccf
 
       auto rv = jsonrpc::pack(rep.value(), ctx.pack.value());
 
-#ifdef PBFT
-      if (history)
-        history->add_response(reqid, rv);
+      return rv;
 #endif
+    }
 
-      return rv;
+    /** Process a serialised command with the associated RPC context via PBFT
+     *
+     * If an RPC that requires writing to the kv store is processed on a
+     * follower, the serialised RPC is forwarded to the current network leader.
+     *
+     * @param ctx Context for this RPC
+     * @param input Serialised JSON RPC
+     */
+    std::vector<uint8_t> process_pbft(
```

Comment:
It's really unfortunate that we have a spit here, we should refactor process() to be common.

Reply:
As discussed, this will be addressed in further PRs.